app/src/main/java/io/appium/uiautomator2/utils/ElementLocationHelpers.java

```diff
@@ -57,10 +67,15 @@ public static String rewriteIdLocator(By.ById by) {
             // transform "textToBeChanged" into:
             // com.example.android.testing.espresso.BasicSample:id/textToBeChanged
             // it's prefixed with the app package.
-            locator = String.format("%s:id/%s", AppiumUIA2Driver
-                    .getInstance()
-                    .getSessionOrThrow()
-                    .getCapability("appPackage"), by.getElementLocator());
+            String packageName = getPackageName();
+            if (packageName == null) {
+                throw new UiAutomator2Exception(String.format(
+                        "Cannot rewrite element locator '%s' to its complete form, because " +
+                                "the current application package name is unknown. Consider " +
```

Comment:
Is this indentation intentional? It is different than the one in the next file changed.

Reply:
its the IDEA's auto formatter ;)